server/server.go

```diff
@@ -742,7 +729,7 @@ func (s *Server) createClient(conn net.Conn) *client {
 	c.Debugf("Client connection created")
 
 	// Send our information.
-	c.sendInfo(info)
+	c.sendInfo(c.generateClientInfoJSON(info))
```

Comment:
Move generation of info json under s.mu.Lock() above.

Reply:
Looking now.

Reply:
Run it over a thousand times locally and nothing.. lol

Reply:
It should be passed by value no?

Reply:
Should already be a copy which I get under the lock from above.

Reply:
Its info.ClientConnectURLs that is getting updated. Will make a more formal copy.

Reply:
Ok, you are doing the copy under the server lock, that's good.